tests/client/search/containers/TestAddonPage.js

```diff
@@ -57,8 +62,13 @@ describe('AddonPage', () => {
         },
       },
     };
+    let root;
+    let info;
 
-    const root = render({state: initialState, props: {params: {slug: 'my-addon'}}});
+    beforeEach(() => {
+      root = render({state: initialState, props: {params: {slug: 'my-addon'}}});
+      info = Array.from(root.querySelector('.addon--info').childNodes);
```

Comment:
The `dataBar` helper in `AddonPage`. https://github.com/mozilla/addons-frontend/blob/c7a3d3bd278d761f6bcfb5335678960240fb4215/src/search/containers/AddonPage/index.js#L59